ci/README.md

```diff
@@ -0,0 +1,134 @@
+This job configures the hosts used for running the jobs.
+It is expected that `stf-base-2node` should not be modified unless there are changes to the upstream jobs.
+
+`stf-base` inherits from `stf-base-2node`, and defines the stf-specific parts of the jobs (prepare hosts, build STF images, deploy STF, test STF).
+
+These jobs are [abstract](https://zuul-ci.org/docs/zuul/latest/config/job.html#attr-job.abstract) and cannot be run directly, however, they contain the plumbing that allows the deployment scenario and OCP version to be configured.
+
+The scenario (`nightly_bundles`, `local_build`, `local_build-index_deploy`) is selected by passing a `scenario` [var to the job](https://zuul-ci.org/docs/zuul/latest/config/job.html#attr-job.vars).
+The OCP version is selected by changing the nodeset that is use in the job.
+
+The jobs are named to describe the combination of scenario and OCP version that is used in the job.
+The naming convention is `stf-crc-ocp_<version>-<scenario> e.g. `stf-crc-ocp_413-local_build`
+
+## OCP version selection
+
+The OCP version selection is done by specifying the `nodeset` to be used by the job.
+The `nodesets` are defined in `.zuul.yaml`. Each nodeset corresponds to a different version of OCP.
+Each nodeset contains two hosts: `crc` and `controller`.
+All ansible playbooks are run against `controller`.
+
+The rest of this section provides further detail on the OCP version selection, and how it relates to CRC and the deployment topology.
+
+The nodesets select the hosts based on labels in zuul.
+The labels available in zuul are shown on the [RDO Zuul labels tab](https://review.rdoproject.org/zuul/labels).
+
+The labels used for the nodesets are `coreos-crc-extracted-<crc_version>-<size>`.
+The “extracted” CRC describes the way that the job deploys and interacts with CRC.
+
+Usually, CRC is run using the `crc start` command, which created a VM on your host which runs the OCP cloud.
+In Zuul, the provisioned hosts are also virtual machines, so running `crc start` would result in a VM in a VM. This nested virtualisation causes some performance issues.
+
+The `extracted` deployment try to address the performance issues associated with nested virtualisation. The infrastructure is more complicated than nested.
+The `coreos-crc-extracted-...` labels provide a VM with an extracted CRC VM image, so that the CRC VM can be booted directly by the cloud provider. The `crc` VM is not accessed directly, but via a second `controller` VM, on which tests are run. The `stf-base-2node` job includes a network configuration to make sure the controller can communicate with the OCP deployment in CRC. All ansible playbooks should be run against `controller`. The `stf-base-2node` job is a common parent to all the jobs in the repo, and should rarely need an update.
```

Comment:
```suggestion
The `coreos-crc-extracted-...` labels provide a VM with an extracted CRC VM image, so that the CRC VM can be booted directly by the cloud provider. The `crc` VM is not accessed directly, but via a second `controller` VM, on which tests are run. The `stf-base-2node` job includes a network configuration to make sure the controller can communicate with the OCP deployment in CRC.
```

Those last two sentences seem a bit off topic in this paragraph and are already mentioned above.